pkgs/applications/office/vnote/default.nix

```
@@ -1,37 +1,42 @@
 { lib
-, mkDerivation
+, stdenv
 , fetchFromGitHub
 , qmake
 , qtbase
 , qtwebengine
+, qtx11extras
+, wrapQtAppsHook
 }:
 
-mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "vnote";
-  version = "3.13.0";
+  version = "3.15.1";
 
   src = fetchFromGitHub {
     owner = "vnotex";
-    repo = pname;
+    repo = "vnote";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-A0OJzhvHP+muPI8N23zD4RTiyK0m3JGr/3uJ0Tqz97c=";
     fetchSubmodules = true;
```

```suggestion
    fetchSubmodules = true;
    hash = "sha256-A0OJzhvHP+muPI8N23zD4RTiyK0m3JGr/3uJ0Tqz97c=";
```
submodules change the hash, so it usually goes last

---

OK.